styles/indexStyles.css

```diff
@@ -0,0 +1,150 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap');
+
+.grand_container__styles{
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    flex-direction: column;
+    position: relative; 
+    font-family: Lato;   
```

Comment:
Give font fallbacks in case Lato font could not be loaded. Would be enough to give proper family - `sans` or `open-sans`